src/modules/dashboard/components/FeaturedChannelList.tsx

```diff
@@ -0,0 +1,27 @@
+import { useGetChannelDetails } from 'queries';
+import { FC } from 'react';
+import { ChannelDetailsProps } from '../configs';
+import FeaturedChannelListItem from './FeaturedChannelListItem';
+
+type FeaturedChannelListprops = {
+  channel: ChannelDetailsProps;
+};
+
+const FeaturedChannelList: FC<FeaturedChannelListprops> = (props) => {
```

Comment:
```suggestion
const FeaturedChannelsList: FC<FeaturedChannelsListprops> = (props) => {
```
@abhishek-01k also change the name of file

Reply:
This was trial file I have to delete this file afaik.